extensions/guacamole-auth-sso/modules/guacamole-auth-sso-openid/src/main/java/org/apache/guacamole/auth/openid/conf/ConfigurationService.java

```diff
@@ -125,6 +133,15 @@ public class ConfigurationService {
         public String getName() { return "openid-groups-claim-type"; }
 
     };
+    /**
+     * The claims within any valid JWT that should be mapped to
+     * the authenticated user's tokens, as configured with guacamole.properties.
+     */
+    private static final StringListProperty OPENID_ATTRIBUTES_CLAIM_TYPE =
+            new StringListProperty() {
+                @Override
+                public String getName() { return "openid-attributes-claim-type"; }
+            };
```

Comment:
This is a very minor thing, but please follow the established style throughout the prior property declarations, particularly as it relates to indentation.